ci-operator/jobs/maistra/istio/maistra-istio-maistra-2.6-presubmits.yaml

```diff
@@ -247,6 +247,7 @@ presubmits:
     - ^maistra-2\.6-
     cluster: build05
     context: ci/prow/integ-servicemesh-2-6
+    optional: true
```

Comment:
This has no effect because we are configuring GH branch settings manually, directly in GH website.